bin/metrics-docker-container.rb

```diff
@@ -46,23 +61,31 @@ class DockerContainerMetrics < Sensu::Plugin::Metric::CLI::Graphite
          default: '/sys/fs/cgroup'
 
   option :docker_host,
-         description: 'Docker host. TCP: "tcp://host:port" or Unix: "unix:///path/to/docker.sock" (default: "tcp://127.0.1.1:2376")',
+         description: 'Docker API URI. https://host, https://host:port, http://host, http://host:port, host:port, unix:///path
+         Default: ENV[DOCKER_URL] => ENV[DOCKER_HOST] => /var/run/docker.sock',
          short: '-H DOCKER_HOST',
-         long: '--docker-host DOCKER_HOST',
-         default: 'tcp://127.0.1.1:2376'
```

Comment:
any reason for completely removing the defaults?

Reply:
The default host is defined in [DockerApi class](https://github.com/rockandska/sensu-plugins-docker/blob/1590252ca5f25826c4684079df5b65e7b770bf64/lib/sensu-plugins-docker/client_helpers.rb#L7) and it is the case for all metrics / checks.
This choice was to prevent to have different default host defined in the future and have a centralized place to define it if there is a need to change it in the future.
So define a default host inside the scripts themselves is not required anymore.

Reply:
fair enough, I think it might be a good idea to call out the precedence in the readme.